config/features/flags.go

```diff
@@ -185,6 +185,12 @@ var (
 		Usage: "Forces the head of the beacon chain to a specific block root. Values can be 'head' or a block root." +
 			" The block root has to be known to the beacon node and correspond to a block newer than the current finalized checkpoint.",
 	}
+	// blackListRoots is a flag for blacklisting block roots from gossip and
+	// downscore peers that send them.
+	blackListRoots = &cli.StringSliceFlag{
+		Name:  "blacklist-roots",
+		Usage: "A comma-separatted list of 0x-prefixed hexstrings. Declares blocks with the given blockroots to be invalid. It downscores peers that send these blocks.",
```

Comment:
nitpick: typo in `separatted`, and `It downscores peers that send these blocks.` is a little clunky. 
Suggestion: `Block roots to reject as invalid, specified as a comma-separated list of  0x-prefixed hex strings. Peers that provide blocks with these roots will be downscored.`